packages/amplify-app/src/xcodeHelpers.js

```diff
@@ -0,0 +1,50 @@
+const xcode = require('xcode');
+const path = require('path');
+const fs = require('fs-extra');
+
+async function getXcodeProjectDir() {
+  const EXTENSION = '.xcodeproj';
+  const files = fs.readdirSync(process.cwd());
+  const targetFiles = files.filter(function(file) {
+    return path.extname(file).toLowerCase() === EXTENSION;
+  });
+  let projDir;
+  if (targetFiles.length) {
+    projDir = path.join(path.join(process.cwd(), targetFiles[0]), '/project.pbxproj');
+  } else {
+    projDir = undefined;
+  }
+  return projDir;
+}
+
+async function addFileToXcodeProj(file) {
+  const projectPath = await getXcodeProjectDir();
+  // Silently return if not in same directory as xcode project
+  if (!projectPath) {
+    return;
+  }
+  const myProj = xcode.project(projectPath);
+  return new Promise((resolve, reject) =>
+    myProj.parse(function(err) {
```

Comment:
cant we use `async`/`await` here?

Reply:
Not possible to change this function to async